config/locales/en.yml

```diff
@@ -391,570 +408,624 @@ en:
+          text: Find out if you’re eligible for Universal Credit
+          href: https://www.gov.uk/universal-credit/eligibility
+        - id: '0029'
+          text: Find out if you can get help from a foodbank (Citizens Advice)
+          href: https://www.citizensadvice.org.uk/wales/benefits/help-if-on-a-low-income/using-a-food-bank/
+          show_to_nations:
+          - Wales
+        - id: '0111'
+          text: Find out if you can get help from a foodbank (Citizens Advice)
+          href: https://www.citizensadvice.org.uk/benefits/help-if-on-a-low-income/using-a-food-bank/
+          show_to_nations:
+          - England
+        - id: '0030'
+          text: Find out about the financial support and benefits you might be able
+            to get (NI direct)
+          href: https://www.nidirect.gov.uk/articles/coronavirus-covid-19-and-benefits
+          show_to_nations:
+          - Northern Ireland
+        - id: '0113'
+          text: If you need food urgently and have no other support, find out if you
+            can get help from your council
+          href: https://www.gov.uk/coronavirus-local-help
+        - id: '0031'
+          text: If you have a child, find out if they can get free school meals
+          href: https://www.gov.uk/apply-free-school-meals
+        - id: '0032'
+          text: Find out if you can apply for Healthy Start vouchers if you’re 10
+            or more weeks pregnant or have a child under 4
+          href: https://www.healthystart.nhs.uk/healthy-start-vouchers/do-i-qualify/
+          show_to_nations:
+          - England
+          - Wales
+          - Northern Ireland
+        - id: '0033'
+          text: Find out if you can apply for Best Start Grant and Best Start Foods
+            if you have a child (mygov.scot)
+          href: https://www.mygov.scot/best-start-grant-best-start-foods/
+          show_to_nations:
+          - Scotland
+        - id: '0034'
+          text: Find out if you’re eligible for the Scottish Welfare Fund (mygov.scot)
+          href: https://www.mygov.scot/scottish-welfare-fund/
+          show_to_nations:
+          - Scotland
+        - id: '0035'
+          text: Find out if you can get a grant if you’re a parent or looking after
+            a child (mygov.scot)
+          href: https://www.mygov.scot/best-start-grant-best-start-foods/
+          show_to_nations:
+          - Scotland
+        - id: '0036'
+          text: Check if you’re eligible for the Discretionary Assistance Fund (Welsh
+            Government)
+          href: https://gov.wales/discretionary-assistance-fund-daf
+          show_to_nations:
+          - Wales
+        - id: '0037'
+          text: Call the coronavirus Community Helpline for Northern Ireland on <a
+            href="tel:+448088020020" class="govuk-link">0808 802 0020</a>, email <a
+            href="mailto:covid19@adviceni.net" class="govuk-link">covid19@adviceni.net</a>
+            or text ‘ACTION’ to 81025
```

Comment:
```suggestion
          text: |
            Call the coronavirus Community Helpline for Northern Ireland on <a
            href="tel:+448088020020" class="govuk-link">0808 802 0020</a>, email <a
            href="mailto:covid19@adviceni.net" class="govuk-link">covid19@adviceni.net</a>
            or text ‘ACTION’ to 81025
```

Reply:
See here for why i think we shouldn't change these: https://github.com/alphagov/govuk-coronavirus-find-support/pull/222/commits/b0c5b0aafdd33f6b983cf66d96c9d34c0e9bfcf6#r430281216